gx_metabolomics.py

```diff
@@ -0,0 +1,94 @@
+import great_expectations as gx
+from great_expectations.data_context import FileDataContext
+
+context = FileDataContext.create(project_root_dir="great_expectations")
+
+from expectations.expect_column_values_to_have_list_length import (
+    ExpectColumnValuesToHaveListLength,
+)
+
+test_dataset = "./metabolomics.json"
+context = gx.get_context()
+validator = context.sources.pandas_default.read_json(test_dataset)
+
+# ad_diagnosis_p_value
+validator.expect_column_values_to_be_of_type("ad_diagnosis_p_value", "list")
+validator.expect_column_values_to_not_be_null("ad_diagnosis_p_value")
+# for custom and experimental expectations you have to pass args as kwargs
+validator.expect_column_values_to_have_list_length(
+    column="ad_diagnosis_p_value", list_length=1
+)
+
+# associated gene name
+validator.expect_column_values_to_be_of_type("associated_gene_name", "str")
+validator.expect_column_values_to_not_be_null("associated_gene_name")
+validator.expect_column_value_lengths_to_be_between(
+    "associated_gene_name", min_value=1, max_value=25
```

Comment:
Jess's validation script has these values: `min=2, max=100`
I haven't checked what our maximum gene name length actually is -- is 25 a high enough maximum?

Reply:
My choices here were just based on the min and max present in the example dataset - I'm happy to change this if need be!

Reply:
I updated the values to match the examples @JessterB shared.